web/app/components/results/index.hbs

```diff
@@ -1,61 +1,22 @@
-{{#if (and this.firstPageIsShown this.queryIsProductName)}}
-  <div
-    class="flex w-full flex-col items-start pb-10"
-    data-test-results-product-link
-  >
-    <Hds::Card::Container
-      @level="mid"
-      @hasBorder="true"
-      @overflow="hidden"
-      class="flex flex-col items-start space-y-3 px-4 pt-4 pb-3"
-    >
-      <Hds::Badge
-        @text={{this.capitalizedQuery}}
-        @icon={{or (get-product-id @query) "folder"}}
-      />
-      <Hds::Link::Standalone
-        @text="View all {{this.capitalizedQuery}} documents"
-        @icon="arrow-right-circle"
-        @iconPosition="trailing"
-        @route="authenticated.documents"
-        @query={{hash product=(array this.capitalizedQuery)}}
-      />
-    </Hds::Card::Container>
-  </div>
-{{/if}}
```

Comment:
This isn't doing anything the search popover isn't already doing, and it takes up a ton of space. I have plans for this in the "All search results" view that will allow us to show all matches instead of 1.